components/sync/driver/BUILD.gn

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//brave/build/features.gni")
+import("//brave/components/sync/driver/sources.gni")
+
+source_set("brave_profile_sync_service_unit_test") {
+  testonly = true
+
+  public_deps = [
+    "//testing/gmock",
+    "//testing/gtest",
+  ]
+
+  sources = [
+    "//brave/components/sync/driver/brave_sync_profile_sync_service_unittest.cc",
+  ]
+
+  sources += brave_components_sync_driver_sources
+
+  defines = brave_service_key_defines
+
+  deps = [
+    "//base",
+    "//brave/components/brave_sync:crypto",
+    "//brave/components/brave_sync:network_time_helper",
+    "//brave/components/brave_sync:prefs",
+    "//components/prefs:prefs",
+    "//components/sync/driver:driver",
+    "//services/network/public/cpp:cpp",
+  ]
+
+  deps += brave_components_sync_driver_deps
```

Comment:
per https://github.com/brave/brave-core/pull/7239/commits/4519f804d8e8422299716947480356206981937c#r533753262, this can also be removed
If we need any from it, please list it directly in deps

Reply:
fixed at https://github.com/brave/brave-core/compare/4519f804d8e8422299716947480356206981937c..4348bf244d9c62618c9632328e1554aec805badd